### ISSUE TITLE
Trouble with reading polish word "Nabory"

### ISSUE THREAD

**Issue by K4cper925** [nvaccess/nvda#17079]
### Steps to reproduce:
You can edit any polish sentence on the webpage by adding word "naobry".  (HTML must have lang="pl" attribute) Then, try to read it. 

### Actual behavior:
NVDA does not read the word “NABORY” correctly in Polish

### Expected behavior:
NVDA read the word “NABORY” correctly in Polish

#### NVDA version:
2024.3.0.33433
#### Windows version:
10 Business

#### Name and version of other software in use when reproducing the issue:
Chrome
#### Other information about your system:
None
#### Does the issue still occur after restarting your computer?
Yes, it can be reporduced on other computers
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
There is no addons

**Comment by zstanecic** [comment]
Hi,

Can you provide more details?

Speech synthesizer used?

W dniu 29.08.2024 o 13:53, Kacper pisze:

**Comment by XLTechie** [comment]
Can you use the voice dictionary to fix this? That is what it is meant to do, since this is likely a problem of the synth's pronunciation, and may not be the case for all voices.

**Comment by seanbudd** [comment]
Closing - this needs to be reported to the speech synthesizer you are using, or fixed locally using your speech dictionary
